packages/material-ui/src/Collapse/Collapse.js

```diff
@@ -39,18 +40,20 @@ export const styles = (theme) => ({
  * [Vertical Stepper](/components/steppers/#vertical-stepper) StepContent component.
  * It uses [react-transition-group](https://github.com/reactjs/react-transition-group) internally.
  */
-const Collapse = React.forwardRef(function Collapse(props, ref) {
+const Collapse = React.forwardRef(function Collapse(props, forwardedRef) {
```

Comment:
To stick the naming convention of the other components?
```suggestion
const Collapse = React.forwardRef(function Collapse(props, ref) {
```

Reply:
done